### PR TITLE
Fix for importing

### PR DESCRIPTION
**Pull request by dnfield**
Relative imports like this won't work in other build systems.  Also minor updates to READMEs

### CODE REVIEW

**Comment by jonahwilliams** [comment]
LGTM (but not to CI)

File: packages/vector_graphics_compiler/test/end_to_end_test.dart [hunk 1]
```diff
@@ -458,8 +458,8 @@ class OnRadialGradient {
         focalX,
         focalY,
         Object.hashAll(colors),
-        Object.hashAll(offsets ?? []),
-        Object.hashAll(transform ?? []),
+        Object.hashAll(offsets ?? Float32List(0)),
```
Comment (jonahwilliams) [inline packages/vector_graphics_compiler/test/end_to_end_test.dart:461]: This isn't a const object so it will be different for the otherwise equal object

**Review by jonahwilliams — CHANGES_REQUESTED** [review]
hashCode change is invalid

File: packages/vector_graphics_compiler/test/end_to_end_test.dart (hunk 1 above)
Comment (dnfield) [reply packages/vector_graphics_compiler/test/end_to_end_test.dart:461]: fiddlesticks

Comment (dnfield) [reply packages/vector_graphics_compiler/test/end_to_end_test.dart]: (fixed it to use `<double>[]`)

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Pull request merged**